.github/workflows/collect_benchmarks.yml

```diff
@@ -0,0 +1,42 @@
+name: Collect benchmarks
+
+on:
+  workflow_run:
+    workflows: ["Run benchmarks main"]
+    types:
+      - completed
+
+concurrency:
+  group: ${{ github.workflow }}
```

Comment:
This bit about concurrency ensures that only one of these "Collect benchmarks" workflows runs at the same time. This is important since concurrent workflows would operate on the same files (could result in duplicated or lost data).